deepcell_label/blueprints.py

```diff
@@ -64,6 +65,24 @@ def raw(token, channel, frame):
     return send_file(png, mimetype='image/png')
 
 
+@bp.route('/api/raw/<token>', methods=['POST'])
+def add_raw(token):
+    """Add new channel to the project."""
+    project = Project.get(token)
+    if not project:
+        return abort(404, description=f'project {token} not found')
+    # Load channel from first array in attached file
+    npz = np.load(request.files.get('file'))
+    channel = npz[npz.files[0]]
+    # Check channel is the right shape
+    expected_shape = (project.num_frames, project.width, project.height, 1)
+    if channel.shape != expected_shape:
+        raise ValueError(f'New channels for project {token} must have shape {expected_shape}')
+    # Add channel to project
+    project.add_channel(channel)
+    return project.make_first_payload()
```

Comment:
Just curious - why `make_first_payload`? We've already sent a first payload before this, right?

Reply:
We send `numChannels` in the first payload and that changes when we add a channel. I was thinking this could be useful in the future to automatically load the new channel if calling the add channel route from the client. I'll make this more compact to only contain the `numChannels` so it's clear that it is different and useful.